aten/src/ATen/cuda/tunable/GemmCommon.h

```diff
@@ -81,7 +82,15 @@ struct GemmParams : OpParams {
   }
 
   std::string Signature() const override {
-    return c10::str(transa, transb, "_", m, "_", n, "_", k);
```

Comment:
Hmm, have you tried `fmt::format`? It should be as fast as `snprintf`

Reply:
Great suggestion to use fmt. Even better performance. Appreciate if you can take a look at the changes to CMake changes.